Description: sparc: use -mcpu=v9 instead of -mcpu=supersparc
 This fixes a build failure of libowfat on sparc:
 .
 diet -v -Os gcc -g -o t t.o libowfat.a `cat libsocket` -lpthread
 gcc -nostdlib -static -L/usr/lib/diet/lib-sparc /usr/lib/diet/lib-sparc/start.o -g -o t t.o libowfat.a -lpthread -isystem /usr/lib/diet/include -D__dietlibc__ -Os -mcpu=supersparc /usr/lib/diet/lib-sparc/libc.a -lgcc /usr/lib/diet/lib-sparc/libc.a
 /usr/lib/diet/lib-sparc/libc.a(vprintf.o): In function `vprintf':
 vprintf.c:(.text+0x1c): warning: warning: the printf functions add several kilobytes of bloat.
 t.o: In function `main':
 t.c:(.text+0x30): undefined reference to `__sync_add_and_fetch_4'
 t.c:(.text+0x48): undefined reference to `__sync_add_and_fetch_4'
 t.c:(.text+0x64): undefined reference to `__sync_bool_compare_and_swap_4'
 t.c:(.text+0x80): undefined reference to `__sync_bool_compare_and_swap_4'
 collect2: ld returned 1 exit status
 .
 According to http://www.debian.org/releases/stable/sparc/ch02s01.html.en
 supersparc isn't supported anymore anyway, and multisparc and niagara
 are v9.
Author: Gerrit Pape <pape@smarden.org>
Forwarded: no
Last-Update: 2010-04-20
---
 diet.c             |    2 +-
 sparc/Makefile.add |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/diet.c
+++ b/diet.c
@@ -30,7 +30,7 @@ static const char* Os[] = {
 	 "-falign-loops=1","-fomit-frame-pointer",0,
   "x86_64","-Os",0,
   "x32","-Os",0,
-  "sparc","-Os","-mcpu=supersparc",0,
+  "sparc","-Os","-mcpu=v9",0,
   "sparc64","-Os","-m64","-mhard-quad-float",0,
   "alpha","-Os","-fomit-frame-pointer",0,
   "arm","-Os","-fomit-frame-pointer",0,
--- a/sparc/Makefile.add
+++ b/sparc/Makefile.add
@@ -1,5 +1,5 @@
 
-CFLAGS+=-mcpu=supersparc -Os
+CFLAGS+=-mcpu=v9 -Os
 CFLAGS+=-DWANT_STRTOD_WITHOUT_LONG_DOUBLE
 VPATH:=sparc:syscalls.s:$(VPATH)
 LIBOBJ+=$(OBJDIR)/udiv.o $(OBJDIR)/umul.o $(OBJDIR)/urem.o
